internal/models/policy/recipe/security/baseline.go

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright © 2022 VMware, Inc. All Rights Reserved.
+SPDX-License-Identifier: MPL-2.0
+*/
+
+package policyrecipesecuritymodel
+
+import "github.com/go-openapi/swag"
+
+// VmwareTanzuManageV1alpha1CommonPolicySpecSecurityV1Baseline Input schema for security policy baseline recipe version v1.
+//
+// swagger:model vmware.tanzu.manage.v1alpha1.common.policy.spec.security.v1.Baseline
+type VmwareTanzuManageV1alpha1CommonPolicySpecSecurityV1Baseline struct {
```

Comment:
Where are we calling these security inputs (baseline/custom/strict)? Is it replacing the `Input interface{}` in `VmwareTanzuManageV1alpha1CommonPolicySpec`

cc: @shreyassreenivas

Reply:
Yes

Reply:
How do we read the values of the security inputs (baseline/custom/strict) then? Say, we need to construct the input for baseline receipt, how do we validate if only the values from the baseline struct is populated and not the other.